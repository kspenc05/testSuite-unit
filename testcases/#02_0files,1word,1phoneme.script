#!/bin/sh
echo "THERE" | ./cmulookup cmudict-0.7b