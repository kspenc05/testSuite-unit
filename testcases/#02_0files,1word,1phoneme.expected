DH EH1 R
DONE! (1 words)
